### ISSUE TITLE
Auction house custom links buttons

### ISSUE THREAD

**Issue by GramThanos** [DinoDevs/GladiatusCrazyAddon#255]
> You could also add auction options saved for example:
> 5 buttons
> I click on 1st button and shows me merc lvl100+
> I click on 2nd and shows upgrades
> I click on 3rd and shows for example Gaius
> I click on 4th and shows only purple items
> 
> You could configure these easily accessible buttons as u want

Suggestion from MBT@GladiatusForum.

**Comment by FrutyX** [comment]
I don't think this is possible, too complicated. Creating search options isn't hard, but making it customizable? Nah.

**Comment by GramThanos** [comment]
Custom buttons on the bars can be possible but we will have to provide icons and variables. The user will have to specify the URL and some parameters e.g.

`mod=packages&sh={SH}` for pachages

**Comment by FrutyX** [comment]
![hqdefault](https://user-images.githubusercontent.com/23278552/149040975-04028b49-f8af-47d2-a55c-b54aca444653.jpg)

**Comment by GreatApo** [comment]
> Custom buttons on the bars can be possible but we will have to provide icons and variables. The user will have to specify the URL and some parameters e.g.
> 
> `mod=packages&sh={SH}` for packages

I think this will be extremely useful oppose to the current auction [+] shortcuts

**Comment by FrutyX** [comment]
Hello!

This Issue has been selected to be merged into our Feature Requests & Enhancements overview Issue -  #530.

Please note that this does not change anything, the request is still taken in account, but it's moved for better and cleaner GitHub management.

Thank you for your feedback and ideas! We appreciate them :).
